devel/libanalysis/include/ert/analysis/analysis_table.h

```diff
@@ -66,13 +67,14 @@ typedef struct {
   analysis_initX_ftype           * initX;
   analysis_init_update_ftype     * init_update;
   analysis_complete_update_ftype * complete_update;
+  analysis_module_data_block_vector_ftype * data_blocks;
```

Comment:
No - remove this; as long as possible the analysis module should be stateless - this state is owned an managed by calling scope.